src/tables.ts

```diff
@@ -729,6 +719,62 @@ export class Tables {
   static STAVE_LINE_THICKNESS = 1;
   static RENDER_PRECISION_PLACES = 3;
   static RESOLUTION = RESOLUTION;
+// 1/2, 1, 2, 4, 8, 16, 32, 64, 128
+  // NOTE: There is no 256 here! However, there are other mentions of 256 in this file.
+  // For example, in durations has a 256 key, and sanitizeDuration() can return 256.
+  // The sanitizeDuration() bit may need to be removed by 0xfe.
+  static durationCodes: Record<string, Partial<GlyphPropsNew>> = {
+    '1/2': {
+      stem: false,
+    },
+
+    1: {
+      stem: false,
+    },
+
+    2: {
+      stem: true,
+    },
+
+    4: {
+      stem: true,
+    },
+
+    8: {
+      stem: true,
+      beamCount: 1,
+      stemBeamExtension: 0,
+      codeFlagUp: '\ue240' /*flag8thUp*/,
+    },
+
+    16: {
+      beamCount: 2,
+      stemBeamExtension: 0,
+      stem: true,
+      codeFlagUp: '\ue242' /*flag16thUp*/,
+    },
+
+    32: {
+      beamCount: 3,
+      stemBeamExtension: 7.5,
+      stem: true,
+      codeFlagUp: '\ue244' /*flag32ndUp*/,
+    },
+
+    64: {
+      beamCount: 4,
+      stemBeamExtension: 15,
+      stem: true,
+      codeFlagUp: '\ue246' /*flag64thUp*/,
+    },
+
+    128: {
+      beamCount: 5,
+      stemBeamExtension: 22.5,
+      stem: true,
+      codeFlagUp: '\ue248' /*flag128thUp*/,
+    },
+  };
```

Comment:
Do we need to add a 256?

Reply:
Perhaps but this should be another PR. It is not a porting issue.